ctlr_test.py

```diff
@@ -0,0 +1,9 @@
+import requests
+
+html_file = requests.get('https://www.m24.ru/news/').text
+with open('test.html', 'w', encoding='utf-8') as f:
+    f.write(html_file)
+
```

Comment:
Nice, this should work in most cases. 
However, your source is rather tricky: try mimicking ordinary user requests by applying various headers and waiting for random time periods inbetween.

Reply:
done